bitbot/dice.go

```diff
@@ -0,0 +1,26 @@
+package bitbot
+
+import (
+	"github.com/whyrusleeping/hellabot"
+	"github.com/justinian/dice"
+	"strings"
+	"fmt"
+	
+	log "gopkg.in/inconshreveable/log15.v2"
+)
+var RollTrigger = hbot.Trigger{
+	func(irc *hbot.Bot, m *hbot.Message) bool {
+		return m.Command == "PRIVMSG"
```

Comment:
You're going to want to look for a trigger command. The current pattern for this is to check for the message type (`PRIVMSG`) and then the triggering command. 

Example: `return m.Command == "PRIVMSG" && strings.TrimSpace(m.Content) == "!info"` will activate this trigger if and only if a user sends a PRIVMSG like `!info` to the channel.

Because yours takes arguments, it should at the very least attempt to split the message and check that the 0th element in the resulting slice is `!roll`

Reply:
For some reason when I do that, it doesn't pick up the first trigger, and then crashes bitbot if I pass just `!roll`. Photo attached.

![screen shot 2018-10-05 at 5 14 23 pm](https://user-images.githubusercontent.com/15082726/46560551-300a6b80-c8c2-11e8-8920-e48d8e5e901a.png)
![screen shot 2018-10-05 at 5 14 33 pm](https://user-images.githubusercontent.com/15082726/46560552-300a6b80-c8c2-11e8-9547-fbf923e301e0.png)